lib/htmlbars-plugin/index.js

```diff
@@ -26,7 +26,10 @@ module.exports = class ClassTransformPlugin {
   determineStylesModule(meta) {
     if (!meta || !meta.moduleName) return;
 
-    let name = meta.moduleName.replace(/\.\w+$/, '');
+    let name = meta.moduleName
+      .replace(/\.\w+$/g, '')
+      .replace(/^((?:@[^/]+\/)?[^/]+\/)src\//, (_, pkg) => pkg);
```

Comment:
The module name can be something like

```
@my-scope/my-package/src/ui/...
```

but it needs to be converted to

````
@my-scope/my-package/ui/...
```

Reply:
I thought the `src` stayed in the import path with MU? My understanding was that one of the goals of the whole setup was that import paths would align with actual files on disk in the sort of standard node-y way by default.

Reply:
Oooh. Yes, you're right. I unknowingly incorrectly implemented the styles transform and carried this mistake over.

As seen here:

![image](https://user-images.githubusercontent.com/834636/48574286-191e5680-e90f-11e8-894f-f87852f5ad52.png)